test/metabase/api/search_test.clj

```diff
@@ -698,21 +711,19 @@
           (model-index/add-values! model-index-1)
           (model-index/add-values! model-index-2)
 
+          ;; TODO This is necessary because we've disabled model-index realtime sync due to toucan issues
+          (search/reindex!)
+
           (testing "Indexed entities returned if a non-admin user has full data perms and collection access"
             (mt/with-all-users-data-perms-graph! {(mt/id) {:view-data :unrestricted
                                                            :create-queries :query-builder-and-native}}
-              (is (=? {"Rome"   {:pk_ref         (mt/$ids $municipality.id)
-                                 :name           "Rome"
-                                 :model_id       (:id root-model)
-                                 :model_name     (:name root-model)
-                                 :model_index_id (mt/malli=? :int)}
-                       "Tromsø" {:pk_ref         (mt/$ids $municipality.id)
```

Comment:
Removed this as New Search is more strict, and would not return it. The search filtering logic is tested elsewhere, at an engine level.